eng/pipelines/vmr-sync.yml

```diff
@@ -0,0 +1,38 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - master
+    - release/*
+
+resources:
+  repositories:
+  - repository: vmr
+    type: github
+    name: dotnet/dotnet
+    endpoint: dotnet
+
+parameters:
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to push to
+  type: string
+  default: ''
+
+variables: 
+- ${{ if ne(parameters.vmrBranch, '') }}:
+  - name: VmrBranch
+    value: ${{ parameters.vmrBranch }}
+- ${{ else }}:
+  - name: VmrBranch
+    value: ${{ replace(replace(variables['Build.SourceBranch'], 'refs/heads/', ''), 'refs/pull/', '') }}
+    
+stages:
+- stage: Synchronize_VMR
+  displayName: Synchronize VMR
+  dependsOn: []
+  jobs:
+  - template: /eng/pipelines/templates/jobs/vmr-synchronization.yml
+    parameters:
+      vmrBranch: ${{ variables.VmrBranch }}
```

Comment:
Technically, for single-stage pipelines, you could just have `jobs:` directly, without stages